server/json-schemas/noncommercial-fields-schema.es6

```diff
@@ -3,21 +3,19 @@ module.exports = {
   'type': 'object',
   'properties': {
     'activityDescription': {
-      'default':'',
+      'default': '',
       'type': 'string'
     },
     'locationDescription': {
-      'default':'',
+      'default': '',
       'type': 'string'
     },
     'startDateTime': {
       'default':'',
-      'pattern':'^(19|20)\\d\\d-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3])Z$',
       'type': 'string'
     },
     'endDateTime': {
       'default':'',
-      'pattern':'^(19|20)\\d\\d-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3])Z$',
```

Comment:
Yes, in the bottom of the util file. A regex alone isn't smart enough to know if it's a valid date.